core/src/main/kotlin/com/exactpro/th2/read/db/app/DataBaseReaderConfiguration.kt

```diff
@@ -28,7 +28,9 @@ class DataBaseReaderConfiguration(
     val dataSources: Map<DataSourceId, DataSourceConfiguration>,
     val queries: Map<QueryId, QueryConfiguration>,
     val startupTasks: List<StartupTaskConfiguration> = emptyList(),
-    val publication: PublicationConfiguration = PublicationConfiguration()
+    val publication: PublicationConfiguration = PublicationConfiguration(),
+    val useTransport: Boolean = false,
+    val sessionGroup: String
```

Comment:
I don't think that the single session group should be used for the whole read-db. If we want to add we need to add the ability to set the group per data source.
Otherwise, it doesn't really make sense (at least for me).
@Nikita-Smirnov-Exactpro what do you think about the current implementation and my opinion?